modules/ztp-adding-new-content-to-gitops-ztp.adoc

```diff
@@ -3,7 +3,151 @@
+      policyName: "group-dev-cluster-log-ns" 
+    - fileName: ClusterLogOperGroup.yaml
+      remediationAction: inform 
+      policyName: "group-dev-cluster-log-operator-group" 
+    - fileName: ClusterLogSubscription.yaml
+      remediationAction: inform 
+      policyName: "group-dev-cluster-log-sub" 
+    #Local Storage Operator
+    - fileName: StorageNS.yaml
+      remediationAction: inform 
+      policyName: "group-dev-lso-ns" 
+    - fileName: StorageOperGroup.yaml
+      remediationAction: inform 
+      policyName: "group-dev-lso-operator-group" 
+    - fileName: StorageSubscription.yaml
+      remediationAction: inform 
+      policyName: "group-dev-lso-sub" 
+    # Performance Addon Operator
+    - fileName: PaoSubscriptionNS.yaml
+      remediationAction: inform 
+      policyName: "group-dev-pao-ns" 
+    - fileName: PaoSubscriptionCatalogSource.yaml
+      remediationAction: inform 
+      policyName: "group-dev-pao-cat-source" 
+      spec:
+        image: <image URL here>
+    - fileName: PaoSubscription.yaml
+      remediationAction: inform 
+      policyName: "group-dev-pao-sub" 
+    #These are custom local polices that come from the source-crs directory in the git repo
+    #Elasticsearch Operator
+    - fileName: elasticsearch/ElasticsearchNS.yaml <1>
+      remediationAction: inform 
+      policyName: "group-dev-elasticsearch-ns" 
+    - fileName: elasticsearch/ElasticsearchOperatorGroup.yaml
+      remediationAction: inform 
+      policyName: "group-dev-elasticsearch-operator-group" 
+    #Bell Custom Resources
+    - fileName: bell/apiserver-config.yaml
+      remediationAction: inform 
+      policyName: "group-dev-apiserver-config" 
+    - fileName: bell/disable-nic-lldp.yaml
+      remediationAction: inform 
+      policyName: "group-dev-disable-nic-lldp" 
+----
+<1> Set `fileName` to include the custom CR subfolder from the `/source-crs` parent.
+
+. Commit the `PolicyGenTemplate` change in Git, and then push to the Git repository being monitored by the GitOps ZTP Argo CD application.
+
+. Update the `ClusterGroupUpdate` CR to include the changed `PolicyGenTemplate` and save it as `cgu-test.yaml`.
++
+[source,yaml]
+----
+apiVersion: ran.openshift.io/v1alpha1
+kind: ClusterGroupUpgrade
+metadata:
+  name: custom-source-cr
+  namespace: ztp-group
+spec:
+  managedPolicies:
+    - group-du-sno-v4.14-config-policy
+  enable: true
+  clusters:
+  - cluster1
+  remediationStrategy:
+    maxConcurrency: 2
+    timeout: 240
+----
+
+. Apply the updated `ClusterGroupUpdate` CR by running the following command:
+[source, terminal]
++
+----
+$ oc apply -f cgu-test.yaml
+----
+
+.Verification
+
+. Check that the updates have succeeded by running the following command:
++
+[source, terminal]
+----
+$ oc  get cgu -A
+----
++
+.Example output
++
+[source, terminal]
+----
+NAMESPACE     NAME               AGE   STATE        DETAILS
+ztp-group     custom-source-cr   6s    InProgress   Remediating non-compliant policies
```

Comment:
NS should be ztp-clusters based on your previous example.

Reply:
ok and thanks.